packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/v2/versionDescription.ts

```diff
@@ -8,6 +8,7 @@ export const versionDescription: INodeTypeDescription = {
 	displayName: 'Call n8n Workflow Tool',
 	name: 'toolWorkflow',
 	icon: 'fa:network-wired',
+	iconColor: 'black',
```

Comment:
please move these to `packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/ToolWorkflow.node.ts` instead, and remove these lines from `packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/v1/versionDescription.ts` as well.

Reply:
Nice, moved `icon` and `iconColor` on the base node, seemed to work.